tests/test_material.py

```diff
@@ -21,6 +21,10 @@ def default_material():
 def test_materials_file(example_repo_path):
     return example_repo_path / 'NIST_ruby/single_GE/include/materials.h5'
 
+@pytest.fixture
+def test_material_file_duplicate_atoms():
+    return f'{test_data_dir}/testmat.h5'
```

Comment:
This will throw an error I think, `test_data_dir` needs to be a parameter in this function

Reply:
Fixed it.